blsSignatures/blsSignatures.go

```diff
@@ -240,19 +256,26 @@ func PublicKeyFromBytes(in []byte, trustedSource bool) (PublicKey, error) {
 }
 
 func PrivateKeyToBytes(priv PrivateKey) []byte {
-	return bls12381.NewFr().Set(priv).ToBytes()
+	bytes := new(fr.Element).Set(priv).Bytes()
+	return bytes[:]
 }
 
 func PrivateKeyFromBytes(in []byte) (PrivateKey, error) {
-	return bls12381.NewFr().FromBytes(in), nil
+	return new(fr.Element).SetBytes(in), nil
 }
 
 func SignatureToBytes(sig Signature) []byte {
-	g1 := bls12381.NewG1()
-	return g1.ToBytes(sig)
+	g1 := new(bls12381.G1Affine)
+	g1.Set(sig)
+	bytes := g1.Bytes()
```

Comment:
Same thing here, I think we can just `return sig.Bytes()[:]`

Reply:
Done.